go.mod

```diff
@@ -41,6 +41,7 @@ require (
 	github.com/RBG-TUM/commons v0.0.0-20220406105618-030c095f6a1b
 	github.com/crewjam/saml v0.4.6
 	github.com/golang/mock v1.6.0
+	github.com/google/uuid v1.1.2
```

Comment:
we already have `github.com/satori/go.uuid` in place, can you use

```go
import (
    uuid "github.com/satori/go.uuid"
)
```
instead to avoid including redundant dependencies?